frontend/src/forms/user-profile-form/UserProfileForm.tsx

```diff
@@ -0,0 +1,127 @@
+import LoadingPageButton from '@/components/LoadingButton';
+import { Button } from '@/components/ui/button';
+import {
+  Form,
+  FormControl,
+  FormDescription,
+  FormField,
+  FormItem,
+  FormLabel,
+} from '@/components/ui/form';
+import { Input } from '@/components/ui/input';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useForm } from 'react-hook-form';
+import { z } from 'zod';
+
+const formSchehma = z.object({
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/19#issuecomment-2380414272>#c8c7f2</a><span> - 09/28/2024, 05:29 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The form schema variable name 'formSchehma' is misspelled, which could lead to confusion and potential errors.
</div>
<div id="fix">
<b>Fix:</b> Correct the spelling of the variable name from 'formSchehma' to 'formSchema'.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -13,14 +13,14 @@
  import { useForm } from 'react-hook-form';
  import { z } from 'zod';
 
 -const formSchehma = z.object({
 +const formSchema = z.object({
    email: z.string().optional(),
    name: z.string().min(1, 'Name is required'),
    addressLine1: z.string().min(1, 'Address Line 1 is required'),
    city: z.string().min(1, 'City is required'),
    country: z.string().min(1, 'Country is required'),
  });
 
 -type UserFormData = z.infer<typeof formSchehma>;
 +type UserFormData = z.infer<typeof formSchema>;
 
  type Props = {
    onSave: (userProfileData: UserFormData) => void;
 @@ -30,7 +30,7 @@
 
  const UserProfileForm = ({ onSave, isLoading }: Props) => {
    const form = useForm<UserFormData>({
 -    resolver: zodResolver(formSchehma),
 +    resolver: zodResolver(formSchema),
    });
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged

Reply:
will update this